cfme/tests/configure/test_access_control.py

```diff
@@ -50,6 +50,39 @@ def new_role(appliance, name=None):
         vm_restriction='None')
 
 
+@pytest.fixture(scope='module')
+def new_tenant_admin(appliance, request):
+    tenant_collection = appliance.collections.tenants
+    child_tenant = tenant_collection.create(
+        name='child_tenant{}'.format(fauxfactory.gen_alphanumeric()),
+        description='tenant description',
+        parent=tenant_collection.get_root_tenant()
+    )
+    request.addfinalizer(child_tenant.delete_if_exists)
+
+    role = appliance.collections.roles.instantiate(name='EvmRole-tenant_administrator')
+    tenant_role = role.copy()
+    with update(tenant_role):
```

Comment:
This Note should be added here :

 Note: BZ 1278484 - tenant admin role has no permissions to create new roles,
            Workaround is to add modify permissions to tenant_administrator role or Roles
            must be created by superadministrator.

Reply:
Done